callbacks/control_bar.py

```diff
@@ -0,0 +1,74 @@
+from dash import Input, Output, State, callback, Patch, MATCH, ALL, ctx
+import dash_mantine_components as dmc
+import json
+
+
+@callback(
+    Output("image-viewer", "figure", allow_duplicate=True),
+    Input("annotation-opacity", "value"),
+    prevent_initial_call=True,
+)
+def annotation_opacity(opacity_value):
+    patched_figure = Patch()
+    patched_figure["layout"]["newshape"]["opacity"] = opacity_value
+    return patched_figure, 3
+
+
+@callback(
+    Output("image-viewer", "figure", allow_duplicate=True),
+    Input("paintbrush-width", "value"),
+    prevent_initial_call=True,
+)
+def annotation_width(width_value):
+    """
+    This callback is responsible for changing the brush width.
+    """
+    patched_figure = Patch()
+    patched_figure["layout"]["newshape"]["line"]["width"] = 11
+    return patched_figure
+
+
+@callback(
+    Output("image-viewer", "figure", allow_duplicate=True),
+    Output("annotation-class-selection", "className"),
+    Input({"type": "annotation-color", "index": ALL}, "n_clicks"),
+    prevent_initial_call=True,
+)
+def annotation_color(color_value):
+    """
+    This callback is responsible for changing the color of the brush.
+    """
+    color_name = json.loads(ctx.triggered[0]["prop_id"].split(".")[0])["index"]
+    hex_color = dmc.theme.DEFAULT_COLORS[color_name][7]
+    patched_figure = Patch()
+    # patched_figure["layout"]["newshape"]["fillcolor"] = hex_color
```

Comment:
Is there a way to have the fill color for the filled area shapes but not for the lines? Right now when I draw a shape it would be great to have it coloured in. 

<img width="352" alt="Screen Shot 2023-06-20 at 3 12 31 PM" src="https://github.com/mlexchange/mlex_highres_segmentation/assets/25990216/112860cc-b431-432c-a6d1-6a68f1e80771">

Reply:
did you mean something like this?
When using "Draw open freeform" the line is coloured, there is no fill happening even if you close the drawn shape
When using "Draw closed freeform" the insides will now be coloured with the same colour, I added 30% transparency so that user can easily see if they made a mistake in drawing the shape, otherwise if it'd be fully coloured, they would have no way of saying

Reply:
Yes, this looks good to me!